mitodl/ocw-course-hugo-theme
# add the heading shortcode

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-to-hugo/issues/218

#### What's this PR do?
This PR adds the `h` shortcode, which takes a single int parameter to determine the heading level.  This is being added to support using headings inside markdown tables, as shortcodes can be written in without requiring a line break like the standard markdown syntax.

#### How should this be manually tested?
 - Make sure you have `ocw-to-hugo` cloned locally
 - Read the readme and set up this project to run locally in Docker, pointing at your local copy of `ocw-to-hugo` for course data
 - Spin up a site locally for `21m-542-interdisciplinary-approaches-to-musical-time-january-iap-2010` and verify that the table on the Syllabus page now has rendered markdown headings in it

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
A related `ocw-to-hugo` PR can be found here: https://github.com/mitodl/ocw-to-hugo/pull/234

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/111838862-507ee280-88d0-11eb-9aa4-adca0800a931.png)
![image](https://user-images.githubusercontent.com/12089658/111838932-67bdd000-88d0-11eb-9056-9921ac782682.png)
